skl2onnx==1.17.0 breaks pipeline

[Pipeline failure](https://github.com/microsoft/hummingbird/actions/runs/9308461310
) related to new [skl2onnx version ](https://pypi.org/project/skl2onnx/1.17.0/) released 8 hours ago
Unsupported model IR version: 10, max supported IR version: 9

Need to pin for now.

```
=========================== short test summary info ============================
FAILED tests/test_onnxml_binarizer_converter.py::TestONNXBinarizer::test_binarizer_converter_0thresh - onnxruntime.capi.onnxruntime_pybind11_state.InvalidArgument: [ONNXRuntimeError] : 2 : INVALID_ARGUMENT : Failed to load model with error: /onnxruntime_src/onnxruntime/core/graph/model.cc:179 onnxruntime::Model::Model(onnx::ModelProto&&, const onnxruntime::PathString&, const onnxruntime::IOnnxRuntimeOpSchemaRegistryList*, const onnxruntime::logging::Logger&, const onnxruntime::ModelOptions&) Unsupported model IR version: 10, max supported IR version: 9
FAILED tests/test_onnxml_binarizer_converter.py::TestONNXBinarizer::test_binarizer_converter_negthresh - onnxruntime.capi.onnxruntime_pybind11_state.InvalidArgument: [ONNXRuntimeError] : 2 : INVALID_ARGUMENT : Failed to load model with error: /onnxruntime_src/onnxruntime/core/graph/model.cc:179 onnxruntime::Model::Model(onnx::ModelProto&&, const onnxruntime::PathString&, const onnxruntime::IOnnxRuntimeOpSchemaRegistryList*, const onnxruntime::logging::Logger&, const onnxruntime::ModelOptions&) Unsupported model IR version: 10, max supported IR version: 9
FAILED tests/test_onnxml_binarizer_converter.py::TestONNXBinarizer::test_binarizer_converter_posthresh - onnxruntime.capi.onnxruntime_pybind11_state.InvalidArgument: [ONNXRuntimeError] : 2 : INVALID_ARGUMENT : Failed to load model with error: /onnxruntime_src/onnxruntime/core/graph/model.cc:179 onnxruntime::Model::Model(onnx::ModelProto&&, const onnxruntime::PathString&, const onnxruntime::IOnnxRuntimeOpSchemaRegistryList*, const onnxruntime::logging::Logger&, const onnxruntime::ModelOptions&) Unsupported model IR version: 10, max supported IR version: 9
FAILED tests/test_onnxml_imputer_converter.py::TestONNXImputer::test_onnx_imputer_const - onnxruntime.capi.onnxruntime_pybind11_state.InvalidArgument: [ONNXRuntimeError] : 2 : INVALID_ARGUMENT : Failed to load model with error: /onnxruntime_src/onnxruntime/core/graph/model.cc:179 onnxruntime::Model::Model(onnx::ModelProto&&, const onnxruntime::PathString&, const onnxruntime::IOnnxRuntimeOpSchemaRegistryList*, const onnxruntime::logging::Logger&, const onnxruntime::ModelOptions&) Unsupported model IR version: 10, max supported IR version: 9
...
...
```

--- a/setup.py
+++ b/setup.py
@@ -31,8 +31,8 @@ install_requires = [
 ]
 onnx_requires = [
     "onnxruntime>=1.0.0,<1.18.0",
-    "onnxmltools>=1.6.0",
-    "skl2onnx>=1.7.0",
+    "onnxmltools>=1.6.0,<1.12.0",
+    "skl2onnx>=1.7.0,<1.16.0",
 ]
 extra_requires = [
     # The need each for these depends on which libraries you plan to convert from

--- a/setup.py
+++ b/setup.py
@@ -31,8 +31,8 @@ install_requires = [
 ]
 onnx_requires = [
     "onnxruntime>=1.0.0,<1.18.0",
-    "onnxmltools>=1.6.0,<1.12.0",
-    "skl2onnx>=1.7.0,<1.16.0",
+    "onnxmltools>=1.6.0,<=1.12.0",
+    "skl2onnx>=1.7.0,<=1.16.0",
 ]
 extra_requires = [
     # The need each for these depends on which libraries you plan to convert from

